synapse/handlers/e2e_keys.py

```diff
@@ -969,12 +961,16 @@ def _process_other_signatures(self, user_id, signatures):
         return signature_list, failures
 
     @defer.inlineCallbacks
-    def _get_e2e_cross_signing_verify_key(self, user_id, key_type, from_user_id=None):
+    def _get_e2e_cross_signing_verify_key(
+        self, user_id, desired_key_type, from_user_id=None
+    ):
         """Fetch the cross-signing public key from storage and interpret it.
```

Comment:
remember that the first line of a docstring is special: it should be a single pithy summary of what the method does. You can then expand on that in further paragraphs.

So, this one is still wrong. Suggest you instead say:

`Fetch or request the given cross-signing public key`

Then soemhting like: "first we try fetching it from storage. If that fails, ...".